components/generic_modal.tsx

```diff
@@ -139,22 +139,20 @@ export default class GenericModal extends React.PureComponent<Props, State> {
                 <Modal.Header
                     closeButton={true}
                 />
```

Comment:
removing form tag. I don't see any use case for keeping it here. @devinbinnie do you remember on top of your mind a specific reason why the generic modal content is surrounded by a <form> tag?